app/src/main/java/xyz/whereuat/whereuat/ContactUtils.java

```diff
@@ -24,6 +24,13 @@ public static InsertCommand buildInsertCommand(Context context, String name, Str
         return new InsertCommand(context, ContactEntry.TABLE_NAME, null, values);
     }
 
+    public static QueryCommand buildSelectContactByPhone(Context context, String phone,
```

Comment:
I've been following all of these methods with `Command` so `buildSelectContactByPhoneCommand`